packages/governance/test/unitTests/test-paramGovernance.js

```diff
@@ -95,8 +91,14 @@ const setUpGovernedContract = async (zoe, electorateTerms, timer) => {
 
   const governedTerms = {
     main: {
-      [MALLEABLE_NUMBER]: makeGovernedNat(602214090000000000000000n),
-      [CONTRACT_ELECTORATE]: makeGovernedInvitation(invitationAmount),
+      [MALLEABLE_NUMBER]: {
+        type: ParamTypes.Nat,
+        value: 602214090000000000000000n,
```

Comment:
Yeah, one of the reasons I liked `makeGovernedNat`, etc. was because clients of a contract have to build the terms, and it's a lot less messy than making them recreate the structure.

Reply:
It was useful. Not sure this way is so bad though. I thought you [said to remove it](https://github.com/Agoric/agoric-sdk/pull/4831#discussion_r826364937). Are you requesting it back?

Reply:
I think I acquiesced, because I thought it wasn't being used any more but here we see that some clients still have to build the structure. The contract writer knows the answer better, and should help when he can. It's not necessary for a test case, but I'll probably add those helpers when I write contracts.